tests/Tests/100__deploy/100__installation/102__post_install.robot

```diff
@@ -204,3 +219,15 @@ Verify BlackboxExporter Includes Oauth Proxy
     @{containers} =    Get Containers    pod_name=${pod}    namespace=redhat-ods-monitoring
     List Should Contain Value    ${containers}    oauth-proxy
     List Should Contain Value    ${containers}    blackbox-exporter
+
+Email In Addon-Managed-Odh-Parameters Secret Should Be
```

Comment:
`Email In Addon-Managed-Odh-Parameters Secret Should Be` and 'Wait Until Notification Email From Addon-Managed-Odh-Parameters Contains' could be in ODS.robot

Reply:
@rkamble5 are you going to implement this suggestion? I see the conversation closed but no comment. Re-opening it

Reply:
Done